packages/assets-controllers/src/Standards/ERC20Standard.ts

```diff
@@ -56,15 +58,10 @@ export class ERC20Standard {
     // Signature for calling `symbol()`
     const payload = { to: address, data: '0x95d89b41' };
     const result = await this.provider.call(payload);
-
-    const abiCoder = new AbiCoder();
-
+    assertIsStrictHexString(result);
     // Parse as string - treat empty string as failure
     try {
-      const decoded = abiCoder.decode(['string'], result)[0];
-      if (decoded?.length > 0) {
```

Comment:
Are we guaranteed that `decoded` can never be empty anymore?

I guess we might be 🤔

Reply:
hmmm you're right. Perhaps I should revert this to only return if defined?

Reply:
cc @Mrtenz

Reply:
In my opinion yeah - if the result is `0x00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000000` it will still return empty string.